torch/testing/_internal/common_utils.py

```diff
@@ -1828,18 +1828,16 @@ def __init__(self, actual, expected, *, rtol_override=0.0, atol_override=0.0, **
             self.actual, self.expected = self._handle_hybrid_sparse_csr(self.actual, self.expected)
 
     def _handle_hybrid_sparse_csr(self, actual, expected):
```

Comment:
Nit
```suggestion
    def _handle_hybrid_sparse_compressed(self, actual, expected):
```

We are trying to be more explicit with these suffixes to avoid confusion.